### ISSUE TITLE
Feature request: close split window via Ctrl-W, C

### ISSUE THREAD

**Issue by nzbart** [VSCodeVim/Vim#2467]
I opened a vertical split window with Ctrl-W, V, and then attempted to close it with Ctrl-W, C. The split window didn't close as expected.

Steps to reproduce:
1. Open a file for editing.
2. Split the window using Ctrl-W, V.
3. Attempt to close the newly opened split window with Ctrl-W, C. It doesn't work.

**Environment**:

VsCode 1.21.1
Vim emulation 0.11.2
Windows 10

**Comment by yogan** [comment]
I agree that it would be nice if VSCodeVim would offer this out of the box.
However, you can bind this yourself. Edit `keybindings.json` and add this snippet:

```
    {
        "key":     "ctrl+w c",
        "command": "workbench.action.closeActiveEditor",
        "when":    "vim.mode != 'Insert'"
    }
```

**Comment by jpoon** [comment]
This would be super easy to add. As an example see: https://github.com/VSCodeVim/Vim/blob/master/src/actions/commands/actions.ts#L2639.

**Comment by tyru** [comment]
Fixed in #2600 (sorry `Fixes #xxx` comment was wrong...)
